www/content/faq.md

```diff
@@ -0,0 +1,447 @@
+# faq
+
+## Where did the name Roc come from?
+
+<img width="128" alt="The Roc logo, an origami bird" src="https://user-images.githubusercontent.com/1094080/92188927-e61ebd00-ee2b-11ea-97ef-2fc88e0094b0.png">
```

Comment:
Not sure we want to load images from githubusercontent.com, we should have the svg somewhere we can load include in the static assets instead. Or at the very least include the PNG in our static assets. 

Also, I think this looks better if we move it down under the sentence "The Roc programming language is named after [a mythical bird", where it is messes with the content layout a bit.